frontend/src/libs/hooks/use-search/use-search.hook.ts

```diff
@@ -0,0 +1,22 @@
+import { type UseSearchReturn } from '#libs/types/types.js';
```

Comment:
No need to create a separate file for this type, it will only be used in this file

Reply:
done